Source/WebInspectorUI/UserInterface/Views/AuditTestGroupContentView.css

```diff
@@ -28,8 +28,8 @@
 }
 
 .content-view.audit-test-group > section > .audit-test-group > header {
-    margin-top: -1px;
-    border-top: 1px solid var(--border-color);
+    margin-top: 0.0625em;
```

Comment:
Why go from a negative 1px margin to a positive expandable one? I can't foresee a problem, but IMO this won't make much difference for low vision users.

Reply:
Okay, was a typo, it should stay positive.